### PR TITLE
ytfzf: update to 2.4.0

### PR DESCRIPTION
**Pull request by Euro20179**

### CODE REVIEW

File: packages/ytfzf/build.sh [hunk 1]
```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/pystardust/ytfzf
 TERMUX_PKG_DESCRIPTION="A POSIX script that helps you find Youtube videos (without API)"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=2.2
+TERMUX_PKG_VERSION=1:2.4.0
```
Comment (Grimler91) [inline packages/ytfzf/build.sh:5]: Prepended `1:` can be removed, doesn't seem to do do anything

**Review by Grimler91 — CHANGES_REQUESTED** [review]

**Review by 2096779623 — CHANGES_REQUESTED** [review]
Can you squash the commits?

File: packages/ytfzf/ytfzf.patch [hunk 2]
```diff
@@ -1,20 +1,38 @@
 --- a/ytfzf
 +++ b/ytfzf
-@@ -189,7 +189,7 @@
- : "${YTFZF_URL_HANDLERS_DIR:=$YTFZF_CONFIG_DIR/url-handlers}"
+@@ -168,7 +168,7 @@
+ 	# clean up only as parent process
+ 	#kill ytfzf sub process{{{
+ 	#i think this needs to be written to a file because of sub-shells
+-	jobs_file="${TMPDIR:-/tmp}/the-jobs-need-to-be-written-to-a-file.list"
++	jobs_file="${TMPDIR:-@TERMUX_PREFIX@}/the-jobs-need-to-be-written-to-a-file.list"
```
Comment (licy183) [inline packages/ytfzf/ytfzf.patch:8]: `/tmp` should be replaced to `@TERMUX_PREFIX/tmp`.

File: packages/ytfzf/ytfzf.patch [hunk 3]
```diff
@@ -1,20 +1,38 @@
...
+ 	jobs -p > "$jobs_file"
+ 	while read -r line; do
+ 		[ "$line" ] && kill "$line" 2> /dev/null
+@@ -312,7 +312,7 @@
  : "${YTFZF_CUSTOM_THUMBNAILS_DIR:=$YTFZF_CONFIG_DIR/thumbnails}"
+ : "${YTFZF_EXTENSIONS_DIR:=$YTFZF_CONFIG_DIR/extensions}"
  
 -: "${YTFZF_SYSTEM_ADDON_DIR:=/usr/local/share/ytfzf/addons}"
-+: "${YTFZF_SYSTEM_ADDON_DIR:=@TERMUX_PREFIX@/share/ytfzf/addons}"
++: "${YTFZF_SYSTEM_ADDON_DIR:=@TERMUX_PREFIX@/local/share/ytfzf/addons}"
```
Comment (licy183) [inline packages/ytfzf/ytfzf.patch:17]: `/usr/local/a` should be replaced with `@TERMUX_PRFIX/a`. I think `/local` is not necessary.

File: packages/ytfzf/ytfzf.patch [hunk 4]
```diff
@@ -1,20 +1,38 @@
...
  
- : "${thumbnail_viewer:=ueberzug}"
+@@ -325,7 +325,7 @@
  
--: "${w3mimgdisplay_path:=/usr/lib/w3m/w3mimgdisplay}"
-+: "${w3mimgdisplay_path:=@TERMUX_PREFIX@/lib/w3m/w3mimgdisplay}"
+     #save the ecurrent environment so that any user set variables will be saved
+     if [ "$check_exists" -eq 1 ]; then
+-	tmp_env=/tmp/ytfzf-env
++	tmp_env=@TERMUX_PREFIX@/ytfzf-env
```
Comment (licy183) [inline packages/ytfzf/ytfzf.patch:26]: `/tmp` should be replaced to `@TERMUX_PREFIX/tmp`.

**Review by licy183 — CHANGES_REQUESTED** [review]

**Review by 2096779623 — APPROVED** [review]

**Comment by Grimler91** [comment]
Thanks!

**Pull request merged**